doc/src/manual/code-loading.md

```diff
@@ -179,7 +179,7 @@ If, on the other hand, Julia was loading the *other* `Priv` package—the one wi
 1. `/home/me/.julia/packages/Priv/HDkrT`
 2. `/usr/local/julia/packages/Priv/HDkrT`
 
-Julia uses the first of these that exists to try to load the public `Priv` package, either from the file `HDkrT`, or the file `HDKrT/src/Priv.jl`.
+Julia uses the first of these that exists to try to load the public `Priv` package from the file `packages/Priv/HDKrT/src/Priv.jl` in the depot where it was found.
```

Comment:
OR the file `packages/Priv/HDKrT`—or is the intent to correct the code to match?